### PR TITLE
Fix use of logicalType in avro schema

### PR DESCRIPTION
**Pull request by a1exsh**
Internal ticket: 1069

### CODE REVIEW

**Comment by adyach** [comment]
👍

File: core-common/src/main/java/org/zalando/nakadi/service/AvroSchema.java [hunk 1]
```diff
@@ -24,12 +24,11 @@
 @Service
 public class AvroSchema {
 
-    public static final byte METADATA_VERSION = 0;
-    public static final Comparator<String> SCHEMA_VERSION_COMPARATOR = Comparator.comparingInt(Integer::parseInt);
+    public static final String METADATA_KEY = "_metadata";
```
Comment (adyach) [inline core-common/src/main/java/org/zalando/nakadi/service/AvroSchema.java:27]: `_` is that indication of the internal_smth?

Comment (adyach) [reply core-common/src/main/java/org/zalando/nakadi/service/AvroSchema.java:27]: to avoid user event type names `metadata`

Comment (a1exsh) [reply core-common/src/main/java/org/zalando/nakadi/service/AvroSchema.java:27]: sort of. it's a temp. solution anyway and hopefully will be replaced with the proper registry

**Pull request merged**